h2o-core/src/main/java/water/api/RequestServer.java

```diff
@@ -665,7 +665,7 @@ private static NanoResponse serveError(H2OError error) {
 
   private static NanoResponse redirectToFlow() {
     NanoResponse res = new NanoResponse(HTTP_REDIRECT, MIME_PLAINTEXT, "");
-    res.addHeader("Location", "/flow/index.html");
+    res.addHeader("Location", H2O.ARGS.context_path + "/flow/index.html");
```

Comment:
i think this is fine but not required.  it might not even be useful.
how would the / request even get there?  the proxy won't route it.

definitely don't prioritize this over getting the required thing to work.  we can always add this later if it turns out we need it.